nifi-nar-bundles/nifi-ignite-bundle/nifi-ignite-processors/src/test/resources/test-default-ignite-client.xml

```diff
@@ -41,4 +37,12 @@
             </bean>
         </property>
     </bean>
+
+    <bean abstract="true" id="thinIgniteClient" class="org.apache.ignite.configuration.ClientConfiguration">
+        <property name="addresses">
+            <list>
+                <value>127.0.0.1:10800</value>
```

Comment:
having statically defined ports in tests almost always is a recipe for spurious failures.  if any other test is running at this time on this port then it will fail.  I'm not sure if there is an option to adjust this but if not we need to ensure none of these tests actually use this port as unit tests - only integration tests.

Reply:
These configurations are only being utilised in Integration Tests.